android/build.gradle

```diff
@@ -74,17 +73,20 @@ repositories {
 dependencies {
     //noinspection GradleDynamicVersion
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.judokit.android:judokit-android:1.0.0'
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.appcompat:appcompat-resources:1.1.0'
     implementation 'com.google.android.gms:play-services-wallet:18.0.0'
-    implementation 'androidx.core:core-ktx:1.2.0'
+    implementation 'androidx.core:core-ktx:1.3.0'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
+//    implementation 'com.judokit.android:judokit-android:1.0.0'
+    implementation 'com.github.Judopay:JudoKit-Android:hotfix~jitpack-io-card-scanning-library-SNAPSHOT'
```

Comment:
Might consider changing this to `develop`

Reply:
yes, it will change but not for now, develop will be gone shortly, we're moving to trunk-based development